cli/flags.go

```diff
@@ -158,9 +160,11 @@ func initFlags(ctx *server.Context) {
 		}
 	}
 
+	isEnvRoachDev := os.Getenv("COCKROACH_DEV") != ""
```

Comment:
i debated putting that detail into the description, but decided that telling the user to set `=1` would be the least confusing